<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMSaveChansDlg</class>
 <widget class="QDialog" name="IMSaveChansDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Which Probe Channels to Save</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="saveChnGB">
     <property name="title">
      <string>Channel list, e.g., '0:3,384:387,768' (Write all as '*' or 'all')</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="3" column="0">
       <widget class="QCheckBox" name="pairChk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Force LF to complement listed AP channels</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Autofill: Only implanted channels from tip upward</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="lenSB">
           <property name="maximum">
            <double>50000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="unitCB">
           <item>
            <property name="text">
             <string>row</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>um from tip</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="setBut">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="rngLbl">
          <property name="text">
           <string>Ranges: AP[0:383], LF[384:767], SY[768].</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="saveChansLE">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>lenSB</tabstop>
  <tabstop>unitCB</tabstop>
  <tabstop>setBut</tabstop>
  <tabstop>saveChansLE</tabstop>
  <tabstop>pairChk</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IMSaveChansDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IMSaveChansDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
